cns/NetworkContainerContract.go

```diff
@@ -404,6 +410,10 @@ type PodIpInfo struct {
 	PodIPConfig                     IPSubnet
 	NetworkContainerPrimaryIPConfig IPConfiguration
 	HostPrimaryIPInfo               HostIPInfo
+	AddressType                     string
+	MacAddress                      string
+	IsDefaultInterface              bool
+	Routes                          []Route
```

Comment:
can you elaborate on what is `IsDefaultInterface` and `Routes`?

Reply:
synced offline